docConfig.json

```diff
@@ -252,16 +252,16 @@
             { "name": "\\$ProductNameShort\\$",     "value": "Ignite UI" },
             { "name": "\\$Platform\\$",             "value": "Blazor"},
             { "name": "{PackageCore}",              "value": "IgniteUI.Blazor"},
-            { "name": "{PackageCharts}",            "value": "IgniteUI.Blazor"},
-            { "name": "{PackageDataSources}",       "value": "IgniteUI.Blazor"},
-            { "name": "{PackageExcel}",             "value": "IgniteUI.Blazor"},
-            { "name": "{PackageGauges}",            "value": "IgniteUI.Blazor"},
-            { "name": "{PackageGrids}",             "value": "IgniteUI.Blazor"},
-            { "name": "{PackageInputs}",            "value": "IgniteUI.Blazor"},
-            { "name": "{PackageLayouts}",           "value": "IgniteUI.Blazor"},
-            { "name": "{PackageMaps}",              "value": "IgniteUI.Blazor"},
-            { "name": "{PackageSpreadsheet}",       "value": "IgniteUI.Blazor"},
-            { "name": "{PackageDockManager}",       "value": "IgniteUI.Blazor"},
+            { "name": "{PackageCharts}",            "value": "Charts"},
+            { "name": "{PackageDataSources}",       "value": "DataSources"},
+            { "name": "{PackageExcel}",             "value": "Excel"},
+            { "name": "{PackageGauges}",            "value": "Gauges"},
+            { "name": "{PackageGrids}",             "value": "Data Grid"},
+            { "name": "{PackageInputs}",            "value": "Inputs"},
+            { "name": "{PackageLayouts}",           "value": "Layouts"},
+            { "name": "{PackageMaps}",              "value": "Maps"},
+            { "name": "{PackageSpreadsheet}",       "value": "Spreadsheet"},
+            { "name": "{PackageDockManager}",       "value": "DockManager"},
```

Comment:
why are you removing "IgniteUI.Blazor" package name and how would customers know package name for Blazor if its name is removed from our docs? Note that these variables also used in other topics (e.g. getting started) where we list actual names of packages that need to be installed by the customers. Maybe, you should do something like this:
```
            { "name": "{PackageCharts}",            "value": "IgniteUI.Blazor (Charts)"},
            { "name": "{PackageGrids}",             "value": "IgniteUI.Blazor (Data Grid)"},
```

Reply:
Also updated Excel IgniteUI.Blazor.Documents.Excel